src/styles/index.scss

```
@@ -1,3 +1,70 @@
+$cell-height: 100px;
+$cell-width: 100px;
+$cell-background-color: #eee;
+$calendar-width: $cell-width * 7 + 6px; // 6px is the total gap between cells
+$days: (
+  mon: 0,
+  tue: 101px,
+  wed: 202px,
+  thu: 303px,
+  fri: 404px,
+  sat: 505px,
+  sun: 606px,
```

This indentation must be calculated inside the loop
```suggestion
  tue: 1,
  wed: 2,
  thu: 3,
  fri: 4,
  sat: 5,
  sun: 6,
```